src/styles/theme.css

```diff
@@ -2,6 +2,12 @@
 @tailwind components;
 @tailwind utilities;
 
+@font-face {
+  font-family: 'Cooper Black';
+  src: url('../assets/coopbl.woff2') format('woff2');
```

Comment:
just a heads up the linter **ls-lint**) is checking the _filename_ itself, so both should match. 👍

Reply:
for some reason git was not registering the change is case on the file so it never got pushed. I will try a new name now.